src/components/homepage-features.module.scss

```diff
@@ -33,6 +33,10 @@
   margin-bottom: 0;
 }
 
+.topics {
```

Comment:
![image](https://user-images.githubusercontent.com/13023439/164003885-4de4a550-f0ee-4aaf-9fff-7b4eb4920c9a.png)
This doesn't seem to work locally (using Chrome 100)